solidity/utils/test_deployment_rsk.js

```diff
@@ -0,0 +1,277 @@
+
+        const newConverter = await converterRegistry.methods.newConverter(type, name, symbol, decimals, '1000000', tokens, weights).call();
+        console.log('called 1 ')
+        await execute(converterRegistry.methods.newConverter(type, name, symbol, decimals, '1000000', tokens, weights));
+        console.log('called 2 ')
+        await execute(converterRegistry.methods.setupConverter(type, tokens, weights, newConverter));
+
+        console.log('New Converter is  ', newConverter);
+
+        console.log('Calling anchors');
+        console.log(await converterRegistry.methods.getAnchors().call())
+
+        const anchor = deployed(web3, 'IConverterAnchor', (await converterRegistry.methods.getAnchors().call()).slice(-1)[0]);
+        const converterBase = deployed(web3, 'ConverterBase', await anchor.methods.owner().call());
+
+        console.log('Now executing the settings on ' +converterBase._address);
+        await execute(converterBase.methods.acceptOwnership());
+        await execute(converterBase.methods.setConversionFee(fee));
+        console.log('Done with conversion fee');
+
+        console.log('Done with ownership acceptance');
+
+        if (type !== 0 && amounts.every(amount => amount > 0)) {
+            for (let i = 0; i < converter.reserves.length; i++) {
+                const reserve = converter.reserves[i];
+                if (reserve.symbol !== 'ETH') {
+                    console.log('Approving amount for ERC20Token')
+                    await execute(deployed(web3, 'ERC20Token', tokens[i]).methods.approve(converterBase._address, amounts[i]));
+                }
+
+                if (type == 2) {
+                    if (!reserve.oracle) {
+                        // can be used to deploy test (static) oracles
+
+                        const chainlinkPriceOracle = await web3Func(deploy, 'chainlinkPriceOracle' + converter.symbol + reserve.symbol, 'ChainlinkETHToETHOracle', []);
+                        reserve.oracle = chainlinkPriceOracle._address;
+
+                    }
+                    await execute(oracleWhitelist.methods.addAddress(reserve.oracle));
+                }
+            }
+
+            if (type == 1) {
+                await execute(deployed(web3, 'LiquidityPoolV1Converter', converterBase._address).methods.addLiquidity(tokens, amounts, 1), 0);
+            } else if (type == 2) {
+                const deployedConverter = deployed(web3, 'LiquidityPoolV2Converter', converterBase._address);
+                await execute(deployedConverter.methods.activate(tokens[0], converter.reserves[0].oracle, converter.reserves[1].oracle));
+
+                for (let i = 0; i < converter.reserves.length; i++) {
+                    console.log('Adding liquidity for LiquidityPoolV2Converter. ', 'For token: ', tokens[i], ' with amount ', amounts[i]);
+                    await execute(deployedConverter.methods.addLiquidity(tokens[i], amounts[i], 1), 0);

```

Comment:
it would be cleaner to keep passing "value" here, but set value to 0 in line 206. Just comment out the original logic of obtaining the value, because we'll need it later.